src/opengl/renderer_ogl.cpp

```diff
@@ -2,12 +2,125 @@
 #include "common/config.h"
 #include "renderer_ogl.h"
 
-namespace dwf {
+#if defined(_WIN32)
+  #include "platform/rendercontext_ogl_win.h"
```

Comment:
Oh, you want indentation here as well.. alright, i'll have to fix those in the vulkan part as well